data/mods/aftershock_exoplanet/region_settings.json

```diff
@@ -0,0 +1,253 @@
+[
+  {
+    "type": "region_settings",
+    "id": "default",
+    "overmap_feature_flag_settings": {
+      "clear_blacklist": false,
+      "blacklist": [ "BLOB", "BEE", "ANT", "FUNGAL", "SLIME", "TRIFFID", "MI-GO", "LAB", "CLASSIC" ],
+      "clear_whitelist": false,
+      "whitelist": [  ]
+    },
+    "default_oter": "field",
+    "default_groundcover": [ [ "t_region_groundcover", 1 ] ],
+    "river_scale": 0,
+    "city": {
+      "shop_radius": 30,
+      "shop_sigma": 80,
+      "park_radius": 20,
+      "park_sigma": 80,
+      "houses": { "afs_city_ruinfield": 400, "afs_formless_ruins_dynamic": 600 },
+      "parks": { "afs_city_ruinfield": 100 },
+      "shops": { "afs_augmentation_clinic_1": 400, "afs_astrobiology_lab": 400 }
+    },
+    "weather": {
+      "base_temperature": -50.0,
+      "base_humidity": 100.0,
```

Comment:
Is there supposed to be continuous precipitation?

Reply:
Have to give the weather a better pass, but yeah. It should snow or at least be cloudy very frequently